CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v1.33.1
```

Comment:
You may be able to unpublish this push in NPM : `npm unpublish <package_name>@<version>`